stream/src/placeholder/handshake.rs

```diff
@@ -95,16 +88,13 @@ impl Handshake {
         // unlike the peer identity) and/or from blocking a peer from connecting
         // to others (if an adversary recovered a handshake message could open a
         // connection to a peer first, peers only maintain one connection per peer).
-        let current_time = runtime
-            .current()
-            .duration_since(UNIX_EPOCH)
-            .expect("failed to get current time")
-            .as_secs();
-        if handshake.timestamp + max_handshake_age.as_secs() < current_time {
-            return Err(Error::InvalidTimestamp);
+        let current_epoch = runtime.current().epoch();
```

Comment:
nit: Although these "are the time since the UNIX epoch", I think it is more common to still just refer to these as "timestamps". I'm not sure if `current_time/current_timestamp -> current_epoch` improves readability.

Reply:
(I'd still leave the function name on `current()` as is)